src/client/stylesheets/style.less

```
@@ -553,4 +553,12 @@ hr.wide {
 	vertical-align: text-top;
 }
 
+.text-collection-entity{
+    text-align: right;
+}
 
+@media only screen and (max-width: 720px){
```

Here we can use the bootstrap variable for screen sizes that is used in other places:
```suggestion
@media only screen and (max-width: @screen-sm-max){
```